crits/core/handlers.py

```diff
@@ -657,6 +657,10 @@ def get_item_names(obj, active=None):
     :returns: :class:`crits.core.crits_mongoengine.CritsQuerySet`
     """
 
+    # Don't use this to get sources.
+    if isinstance(obj, SourceAccess):
+        return []
```

Comment:
Didn't realize that this function is supposed to return a CritsQuerySet. By returning a list I don't know what that would do (I assume break stuff?).